client/src/utils/createGroupedList.js

```diff
@@ -0,0 +1,19 @@
+export const createGroupedList = (list, size) => {
+  if (!size) {
+    console.log('no size provided to createGroupedList');
+    return list;
+  }
+  if (!list) {
+    console.log('no list provided to createGroupedList');
```

Comment:
Same here

Reply:
my concern was I didn't want the function to return early silently.  I replaced the above console logs with errors.